target_xero/__init__.py

```diff
@@ -185,6 +184,9 @@ def post_journal_entries(journals, client):
             # Push the journal entry
             res = client.push("Manual_Journals", journal)
             res = res.json()
+            if "Type" in res and res["Type"] == "ValidationException" and "Elements" in res:
+                #Log validation errors
+                logger.error(f"Journal Entry validation error: {json.dumps(res['Elements'])}")
```

Comment:
if it goes in here, shouldn't we raise an Exception too? I see below it says `don't want to allow a partially successful post`

Reply:
It will be automatically raised when target will try to read `ManualJournalID` at line 191